deepspeed/inference/v2/kernels/ragged_ops/linear_blocked_kv_rotary/blocked_kv_rotary.cpp

```diff
@@ -51,6 +52,8 @@ void kv_trained_rotary_embeddings(torch::Tensor& kv_cache,
     TORCH_CHECK(n_tokens == k.size(0));
     TORCH_CHECK(n_tokens == v.size(0));
 
+    const float theta_base = 0.f;
```

Comment:
Yes, on this code path the theta base is irrelevant since we're reading the buffer from the model that would be derived from it. The only reason we declare it here is compatibility with the macro/single interface for the pair of rotary kernels. As the interfaces have diverged more I have wondered if it makes more sense to just split the interface entirely.